libraries/c_sdk/standard/mqtt/src/private/iot_mqtt_internal.h

```diff
@@ -914,3 +914,146 @@ void _IotMqtt_CloseNetworkConnection( IotMqttDisconnectReason_t disconnectReason
                                       _mqttConnection_t * pMqttConnection );
 
 #endif /* ifndef IOT_MQTT_INTERNAL_H_ */
+
+/*----------------- MQTT Serailization /Deserialization Wrapper functions for Shim------------------*/
```

Comment:
I believe these wrapper functions should not be below the header include guards unless you had a specific reason for doing so.